models/classes/Comment/ItemCommentService.php

```
@@ -0,0 +1,132 @@
+    }
+
+    private function getFeatureFlagChecker(): FeatureFlagCheckerInterface
+    {
+        return $this->getServiceLocator()->get(FeatureFlagChecker::class);
+    }
+
+    private function assertFeatureEnabled(): void
+    {
+        if (
+            !$this->getFeatureFlagChecker()->isEnabled(
+                FeatureFlagCheckerInterface::FEATURE_FLAG_ITEM_COMMENTS_ENABLED
```

PHPStan (2.2.6)

[error] Access to undefined constant oat\tao\model\featureFlag\FeatureFlagCheckerInterface::FEATURE_FLAG_ITEM_COMMENTS_ENABLED.


(classConstant.notFound)